### DISCUSSION TITLE
Content Type with datetime field include seconds

### DISCUSSION THREAD

**Discussion by mybrokengnome** [strapi/strapi#8302] [Legacy Topics]
I'm using Strapi to power an API and have a content type with a datetime field named PostedAt. 

I'm having trouble figuring out how to get this field to let the user input hours, minutes, AND seconds.
![image](https://user-images.githubusercontent.com/161018/95803137-580ce080-0ccd-11eb-96b7-97f777218528.png)

I found one page that mentioned updating this file extensions/content-manager/admin/src/utils/dateFormat.js

So I changed datetime to this format datetime: 'dddd, MMMM Do YYYY HH:mm:ss' in it, but unfortunately this had no affect after a rebuild. The API returns a full datetime, including seconds so this should be possible I would think

![image](https://user-images.githubusercontent.com/161018/95803212-88ed1580-0ccd-11eb-86e7-6aabe78bd11b.png)

Looking at the UI for the content type the time input box is type=text and step=30, and it will let me type in minutes that aren't a 30, i.e. 14:07 for 2pm ET, but won't let me enter 14:07:44.

Thanks in advance!

**Comment by mybrokengnome** [comment]
Still having trouble getting it to work with seconds, but tracked it down some more.

The input box is coming from buffetjs, which does support seconds after looking into the TimePicker/index.js file. It's a bool that gets passed into the props.

So as a test, before I went much further, I located this file node_modules/strapi-plugin-content-manager/admin/src/components/FilterPickerOption/Input.js which seems to be what loads the buffet component. Tried hardcoding seconds=true into the component call and rebuilding, and still nothing. Went a step further and put this file into my extensions... folder and rebuilt again, still didn't affect anything. I'm kind of stumped.

**Comment by derrickmehaffy** [comment]
Hello,

I see you are wanting to ask a question, we are currently planning to decomission our GitHub Discussions in early 2021 and we don't want to lose the great questions asked by our community.

Questions should be directed to [our forum](https://forum.strapi.io/). Please see the following contributing guidelines for asking a question [here](https://github.com/strapi/strapi/blob/master/CONTRIBUTING.md#reporting-an-issue).
